packages/kolibri-common/components/Cards/AccessibleResourceCard.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+
+  <div>
+    <KCard
+      :to="to"
+      :headingLevel="headingLevel"
+      layout="horizontal"
+      thumbnailDisplay="large"
+      :title="contentNode.title"
+      :thumbnailSrc="thumbnailSrc"
+      :thumbnailScaleType="thumbnailScaleType"
+      thumbnailAlign="right"
+      :style="{ height: '172px', width: '500px', margin: '16px 0 16px 0' }"
+    >
+      <template #thumbnailPlaceholder>
+        <div class="default-resource-icon">
+          <LearningActivityIcon :kind="contentNode.learning_activities" />
+        </div>
+      </template>
+      <template #belowTitle>
+        <div>
+          <KTextTruncator
+            :text="contentNode.description"
+            :maxLines="2"
+          />
+        </div>
+      </template>
+      <template #footer>
+        <div class="footer-icon-style">
+          <KIconButton
+            icon="bookmarkEmpty"
+            size="mini"
+            :color="$themePalette.grey.v_600"
+            :ariaLabel="coreString('savedFromBookmarks')"
+            :tooltip="coreString('savedFromBookmarks')"
+            @click="$emit('toggleInfoPanel')"
```

Comment:
Nice, I think at this point it makes sense for this card to just emit on action rather than to contain handling implementation.

**[required]** I think we will need to rename emitted event names though to something meaningful. For example just `toggleBookmark` and for the other emit on line 45 it could be `toggleInfo`? I think this should reflect these two icon buttons.